app/services/create_project_activity.rb

```diff
@@ -23,6 +24,8 @@ def call
     activity.accountable_organisation_reference = "GB-GOV-13"
     activity.accountable_organisation_type = "10"
 
+    activity.extending_organisation = user.delivery_partner? ? user.organisation : nil
```

Comment:
Providing our authorisation code is working as intended, is there a scenario where a user other than a delivery partner is allowed to create a project?

Reply:
Yes, I think all user roles can create projects. The tests in "Users can create a project level activity" show all roles can create projects - is this not supposed to be the case?

Reply:
I was expecting that only delivery partners would ever create projects, modelling what happens now as they're the main reporter.

My thinking has been that BEIS should be able to create funds and programmes whilst DP can only create projects and sub-projects. There's no harm in delivery partners being able to view the hierarchies above for extra context but there's not a clear need for it.

Reply:
OK shall I refactor it do only DPs can create projects? Can other roles view projects?

I'm cognizant that this needs to be done for the project board and I'm off on Monday!

Reply:
I think the story is to enable delivery partners so yes I think so.

> Can other roles view projects?

There is no express need for other roles to view them at the moment. Approval processes will come eventually but that's for later. That said there's probably no impact if they can. I'd go for the simpler and faster option around who can view.